.github/workflows/publish.yml

```diff
@@ -0,0 +1,51 @@
+name: publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Version to Checkout and Release'
+        required: true
+        default: 'v0.1.2'
+
+jobs:
+  publish:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      -
+        uses: actions/checkout@master
+        with:
+          repository: rakyll/hey
+          ref: ${{ github.event.inputs.tag }}
+          fetch-depth: 1
+      -
+        name: Generate Binaries
+        run: |
+          CGO_ENABLED=0 GOOS=linux go build -mod=readonly -o bin/hey && \
+          CGO_ENABLED=0 GOOS=darwin go build -mod=readonly -o bin/hey-darwin && \
+          CGO_ENABLED=0 GOOS=windows go build -mod=readonly -o bin/hey-windows.exe && \
+          CGO_ENABLED=0 GOOS=linux GOARCH=arm GOARM=6 go build -mod=readonly -o bin/hey-armhf && \
+          CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -mod=readonly -o bin/hey-arm64
+      -
+        name: Create release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.event.inputs.tag }}
+          draft: false
+          prerelease: false
+      -
+        name: Upload binaries to release
```

Comment:
Why aren't we using my version of this like in openfaas etc?

Reply:
The action you built works really well for repositories that do not use the, with.repository value.
```yaml
with:
  repository: rakyll/hey
```
But in this case, we have a remote that points to our own repository, and I get this error
<img width="592" alt="Screenshot 2020-12-01 at 5 23 43 PM" src="https://user-images.githubusercontent.com/25264581/100737378-1c071980-33fa-11eb-90bc-a8c4003e5498.png">

Reply:
Can you explain that, I don't understand?

Reply:
If you look at the latest changes, I've temporarliy added one more commit that uses your action, but I'm getting the same error as in the screenshot above 
https://github.com/utsavanand2/hey-redist/runs/1480130018?check_suite_focus=true

Reply:
I'm guessing but I'm not sure why it says Not Found. Is it not able to find the remote of the cloned repo (https://github.com/rakyl/hey.git) the same as hey-redist (https://github.com/utsavanand2/hey-redist.git)?